src/applications/check-in/components/HelpBlock.jsx

```diff
@@ -103,7 +103,7 @@ const HelpBlock = ({ travel }) => {
 };
 
 HelpBlock.propTypes = {
-  travel: PropTypes.bool.isRequired,
+  travel: PropTypes.bool,
```

Comment:
With removing the isRequired, it may be a good idea to set a default value. Probably default false?

Reply:
or maybe the absence is falsy already right...?

Reply:
ok disregard this comment, answered my own question above. All looks good!